mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/226 pause characters

Closes #226

This prevents the characters from animating, pathing, and AI'ing while the simulation is paused. 

To Verify:
- Run the Physical Security scenario (as it has the most characters)
- Hire a security guard or two
- Notice all the characters are frozen because the game is Paused.
- Unpause the game
- Notice the characters start animating and pathing. 
- you should be able to toggle the Play state and see all the characters freeze and resume.

Note, this doesn't freeze the characters when the simulation is Playing and a Dialog is open -- not sure how to detect that.
